### PR TITLE
Disable console

### PR DESCRIPTION
**Pull request by CLIcoder**

### CODE REVIEW

File: src/client.js [hunk 1]
```diff
@@ -1,7 +1,15 @@
+
+// DISABLE CONSOLE LOG IN PRODUCTION
+if (window.location.hostname !== 'localhost'){
+	console.log = function() {}
```
Comment (pavle-goloskokovic) [inline src/client.js:4]: this is not a safe way to disable `console.log`
babel can be instructed to strip `console.log` calls during build process.

**Review by pavle-goloskokovic — CHANGES_REQUESTED** [review]

File: src/client.js (hunk 1 above)
Comment (CLIcoder) [reply src/client.js]: I added the fix

Comment (pavle-goloskokovic) [reply src/client.js]: It's a better approach but it misses other log methods such as `console.warn` or `console.error`.
I was thinking of adding this plugin to babel and have it take care of all of it, or configure it the way it suits us:

https://babeljs.io/docs/en/babel-plugin-transform-remove-console/

**Comment by pavle-goloskokovic** [comment]
> One thing we do rely on occasionally is when the game ends, the server console.logs the last 50 scripts. I'm worried about losing that here.

You can update those specific logs to use `console.info` and then exclude `info` in plugin config. That way you keep what you need and remove all unneeded.

**Review by Grenagar — APPROVED** [review]

**Pull request merged**